### PR TITLE
TWW Services and Treatise

### PR DESCRIPTION
**Pull request by francisalbar**
Add TWW Services and Treatise

### CODE REVIEW

File: SavedInstances/Modules/Progress.lua [hunk 1]
```diff
@@ -1061,6 +1061,63 @@ local presets = {
...
+      [83727] = "Algari Treatise on Enchanting",
+      [83728] = "Algari Treatise on Engineering",
+      [83729] = "Algari Treatise on Herbalism",
+      [83730] = "Algari Treatise on Inscription",
+      [83731] = "Algari Treatise on Jewelcrafting",
+      [83732] = "Algari Treatise on Leatherworking",
+      [83733] = "Algari Treatise on Mining",
+      [83734] = "Algari Treatise on Skinning",
+      [83735] = "Algari Treatise on Tailoring",
+    },
```
Comment (LiangYuxuan) [inline SavedInstances/Modules/Progress.lua:1116]: use locale string, `L["Algari Treatise on Alchemy"]`

File: SavedInstances/Modules/Quest.lua [hunk 2]
```diff
@@ -151,6 +151,26 @@ local _specialQuests = {
...
   [81090] = { name = L["Alchemy Thaumaturgy"] }, -- Alchemy Thaumaturgy
 
+  -- TWW Profession Services
+  [84133] = "Weekly", -- Alchemy Services Requested
+  [84127] = "Weekly", -- Blacksmithing Services Requested
+  [84128] = "Weekly", -- Engineering Services Requested
+  [84129] = "Weekly", -- Inscription Services Requested
+  [84130] = "Weekly", -- Jewelcrafting Services Requested
+  [84131] = "Weekly", -- Leatherworking Services Requested
+  [84132] = "Weekly", -- Tailoring Services Requested
```
Comment (LiangYuxuan) [inline SavedInstances/Modules/Quest.lua:161]: should be put in `QuestExceptions` in this file

**Review by LiangYuxuan — CHANGES_REQUESTED** [review]
Thanks for your contributing! Here is some changes needed before merged.

File: SavedInstances/Modules/Progress.lua (hunk 1 above)
Comment (francisalbar) [reply SavedInstances/Modules/Progress.lua]: Adjusted

File: SavedInstances/Modules/Quest.lua (hunk 2 above)
Comment (francisalbar) [reply SavedInstances/Modules/Quest.lua]: Adjusted, thanks for feedback

**Review by LiangYuxuan — APPROVED** [review]

**Pull request merged**